seqr/views/utils/dataset_utils.py

```diff
@@ -109,34 +110,29 @@ def match_sample_ids_to_sample_records(
         sample_type (string): one of the Sample.SAMPLE_TYPE_* constants
         dataset_type (string): one of the Sample.DATASET_TYPE_* constants
         elasticsearch_index (string): an optional string specifying the index where the dataset is loaded
-        create_sample_records (bool): whether to create new Sample records for sample_ids that
-            don't match existing Sample records, but do match individual_id's of existing
-            Individual records.
         sample_id_to_individual_id_mapping (object): Mapping between sample ids and their corresponding individual ids
         loaded_date (object): datetime object
+        raise_no_match_error (bool): whether to raise an exception if no sample matches are found
+        unmatched_error_template (string): optional template to use to raise an exception if samples are unmatched
```

Comment:
The `unmatched_error_template` is both a condition and a template. To avoid the intuition of just a template, maybe, we can rename it to `raise_unmatched_template`. And, the comments for it can be changed to something like `whether to raise an exception if there are unmatched samples when the template is not None`.